integration/hive/hive-guide.md

```diff
@@ -91,7 +91,6 @@ $HIVE_HOME/bin/hive
 ### Query data from hive table
 
 ```
-alter table hive_carbon set location '<hdfs store path>/hive_carbon';
```

Comment:
why remove this row description?

Reply:
@chenliang613 there is no need of alter table statement to alter location now it is done by CarbonHiveMetastore Event listener itself

Reply:
@chenliang613 this code right here
 val serdeInfo = table.getSd.getSerdeInfo
            serdeInfo.setSerializationLib("org.apache.carbondata.hive.CarbonHiveSerDe")
            val tablePath = serdeInfo.getParameters.get("tablePath")
            if (tablePath != null) {
              table.getSd.setLocation(tablePath)
            }